tests/UriTest.php

```diff
@@ -113,19 +113,11 @@ public function getInvalidUris()
     public function testPortMustBeValid()
     {
         $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessage('Invalid port: 100000. Must be between 1 and 65535');
+        $this->expectExceptionMessage('Invalid port: 100000. Must be between 0 and 65535');
 
         (new Uri())->withPort(100000);
     }
 
-    public function testWithPortCannotBeZero()
-    {
-        $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessage('Invalid port: 0. Must be between 1 and 65535');
-
-        (new Uri())->withPort(0);
```

Comment:
Could you modify this to use -1 as port?

Reply:
That's fixed now.